pages/event/[eventId].tsx

```diff
@@ -35,7 +38,100 @@ const EventPage: NextPage = () => {
     return <>{JSON.stringify({ error })}</>;
   }
 
-  return <>{JSON.stringify(event)}</>;
+  // Calculations for remaining days
+  const daysLeft = () => {
```

Comment:
@franmper para mi esto deberia traerlo el BE y no tener que calcularlo en el FE, cómo lo ves?

Reply:
listo @yolitzr @nykka cuando actualicen main van a tener una nueva propiedad con este dato. Utiliza la misma función que hizo yolit, asi que no deberia cambiar el formato. Cualquier cosa me avisan

Reply:
Genial chixs!!! Ya actualizo esto.